seed/seedwriter/writer.go

```diff
@@ -547,6 +547,12 @@ func (w *Writer) SetInfo(sn *SeedSnap, info *snap.Info) error {
 		return err
 	}
 
+	if info.Confinement == snap.DevModeConfinement {
+		if err := w.policy.allowsDangerousFeatures(); err != nil {
```

Comment:
to improve the error, to put in focus what's the problem is, we could pass a feature in play enum value to this function. Doesn't seem something to do now though

Reply:
yeah as annoying as it is for me personally, most people won't see this unless they see it from ubuntu-image, and even then they will be building their own images which is an infrequent task I think.